client/src/utils.ts

```diff
@@ -126,3 +126,13 @@ export const getSubtotalPrice = (cartItem: CartItem) => {
     PRICE_FRACTION_DIGITS
   );
 };
+
+// Returns a list of strings
+export const parseRawTextNewlines = (text: string): string[] => {
+  return text.split("%0A");
```

Comment:
Whats this symbol corresponding to? Add comment
```suggestion
  ("%0A");
```

Reply:
URL encoded newline character `\n`, added in comments